### ISSUE TITLE
Make the whole participant row click-able 

### ISSUE THREAD

**Issue by nivemaham** [RADAR-base/radar-upload-source-connector#52]
It is less intuitive if the user can only click on the down-arrow to expand the view and upload files

**Comment by khoathanglong** [comment]
I don't think so. It's hard to follow up if expanding too many rows, since the list may be long. Plus, too many rows expanding may requires a large amount of record/file data store in the memory, which may slow down or crash the app.

**Comment by nivemaham** [comment]
I don't understand. If a user wants to expand "too many" rows, they can do that now as well. I am not talking about expanding all rows by default. I am talking about how you can expand a row, is it either by being able to click somewhere else on the row or just on the small arrow.

**Comment by khoathanglong** [comment]
sorry I mis-read it. Yes user can click the row to expand. Clicking to the arrow only is to prevent user from accidentally expanding the row (for example: they just want to copy the data)
